src/goal_sequence_visualizer.py

```diff
@@ -35,8 +35,9 @@ def init_goal_sequence_visualizer(mat):
 		pose.orientation.z = quaternion[2]
 		pose.orientation.w = quaternion[3]
 		pose_array.poses.append(pose)
-		marker.header.frame_id = "/map"
+		marker.header.frame_id = "map"
 		marker.header.seq = i
+		marker.ns = ""
```

Comment:
Not required I guess! I will not have it!

Reply:
🥲ops